### PR TITLE
Proposal 311:Added evasive diction bonus.

### PR DESCRIPTION
**Pull request by rjulian**
A little proposal to add some challenge for those point hungry amongst us! This is essentially the drink, drank, drunk rule from all of the horrible drinking games in college. 

Give me your vote if you favor, long, rich diction that strays away from the direct and efficient and moves us into the downright creative. 

As this rule is optional, I feel it could go to the wayside, but for someone like me, who's sticking to his single digit point values, I really may need the option to pull out a thesaurus so that I can get a little more ahead.

### CODE REVIEW

File: mutable_rules/311_Evasive_diction_bonus.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The play proposing the rule may score additional points if, during discussion (i.e. within comments in a Pull Request), they refrain from using the following strings (case insensitive):
+
+* Rule
+* Vote
+* Score
+* Proposal
+* Repeal
+* Amend
+
+The player receives and additional half of total points earned for a successful proposal. For a defeated proposal, a player receives five points.
```
Comment (audiodude) [inline mutable_rules/311_Evasive_diction_bonus.md]: "and additional" => "an additional"

File: mutable_rules/311_Evasive_diction_bonus.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+The play proposing the rule may score additional points if, during discussion (i.e. within comments in a Pull Request), they refrain from using the following strings (case insensitive):
```
Comment (audiodude) [inline mutable_rules/311_Evasive_diction_bonus.md]: "The play proposing the rule" => "The player proposing a rule"

Comment (audiodude) [inline mutable_rules/311_Evasive_diction_bonus.md]: "they refrain" => "he or she refrains"

**Comment by audiodude** [comment]
I've made some suggested amendments to the grammar of the rule.

I actually think this might be too easy, and might suppress discussion, because you could simply dump a rule change with no comment at all and be done with it, right?

**Comment by rjulian** [comment]
No, because if you did that, it's unlikely we'd vote for the proposal. The discussions are what should be getting you points, not just rule changes. Otherwise, this game would simply be a public opinion poll without grandstanding. You would net -5 points rather than -10, but yet again, that's not going to help you.

Also, god damn, I SwEaR I know the Englishes better than that.

**Comment by sashahart** [comment]
This is funny.

I would vote for this in exchange for a vote for complex-number scoring.

It might be funnier if the evasive diction bonus cannot be awarded unless SOME words other than taboo words were used in comments (no guarantee they related to anything, though). I'm not sure if that was audiodude's idea of saying it is too easy. Maybe this is for the longer term, but if this gets through, I would be in favor of even more bizarrely restrictive criteria, like that you could not use any words used in the last n proposals without losing the evasive diction bonus.

**Comment by audiodude** [comment]
I think you should also add "Transmute" to the list, and it looks like you haven't pushed your changes to update the wording.

I would vote for this in exchange for a vote against complex-number scoring.

**Comment by rjulian** [comment]
@sashahart considering I promised in a previous proposal, you have my vote. Sorry @audiodude. 

+1

**Comment by sashahart** [comment]
+1

**Comment by audiodude** [comment]
-1

**Pull request merged**